perma_web/frontend/components/UploadForm.vue

```diff
@@ -137,8 +142,13 @@ defineExpose({
             "This will create a new Perma Link." }}
```

Comment:
Looks good! One suggestion is to lowercase the l in Link. However, I see both usages in the code base, uppercase and lowercase; so up to you!

Reply:
I always think "Perma Link" looks weird, but it is a term of art in Perma docs, I believe. @clare-stanton has spoken firmly to me about this before.

Reply:
Perma Link, please  :)

Reply:
I think "Perma Link" as a two-word proper noun has always confused my brain a bit. So I understand both of these perspectives — and definitely support the decision to keep it capitalized as a matter of consistent style. Thank you for providing these perspectives, y'all!